doc/md_docs.go

```diff
@@ -25,23 +25,52 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
+// For formatting options and aliases. E.g., ["--help", "-h"] becomes "`--help`, `-h`"
+func formatFlags(input []string) string {
+	return fmt.Sprintf("`%s`", strings.Join(input, "`, `"))
+}
+
+func formatDescription(input string) string {
+	return strings.Replace(input, "\n", "<br/>", -1)
```

Comment:
Alternative would be to replace line changes with spaces, but that would result to different formatting than in `--help` messages. `<br/>` HTML tag should be quite widely supported by markdown parsers.